examples/ingest/google_cloud_storage/ingest.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+# Processes several files in a nested folder structure from gs://unstructured_public/ 
+# through Unstructured's library in 2 processes.
+
+# Structured outputs are stored in gcs-output/
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+cd "$SCRIPT_DIR"/../../.. || exit 1
+
+PYTHONPATH=. ./unstructured/ingest/main.py \
+   --remote-url gs://unstructured_public/ \
+   --structured-output-dir gcs-output \
+   --num-processes 2 \
+   --recursive \
+   --verbose 
```

Comment:
I pushed up a gcs test bucket with private and public paths, we should update to use that. I'll follow up with details directly.

Reply:
let's use gs://utic-test-ingest-fixtures

Reply:
This would require the user to have authentication token. Is that what we really want? Or do we want a no credits example?

Reply:
oh sorry, this is for the example. yes, no auth. I'll see if I can make a public bucket for you